app/models/bulkrax/importer_run.rb

```diff
@@ -5,6 +5,8 @@ class ImporterRun < ApplicationRecord
     belongs_to :importer
     has_many :statuses, as: :runnable, dependent: :destroy
 
-    serialize :parents, Array
+    def parents
```

Comment:
does this get called now with `importer.last_run.parents` in the schedule relationships job?